tests/integration/testlib/iprule.py

```diff
@@ -39,6 +39,7 @@ def ip_rule_exist_in_os(ip_from, ip_to, priority, table):
     found = True
     for rule in current_rules:
         if rule.get('src') == 'all' or rule.get('dst') == 'all':
+            found = False
```

Comment:
I do not fully understand this code. In line 39 and line 49 i`found` is set to `True´ and in line 66 it is checked.

So, if current_rules is empty, found will be True or if any of the conditions on lines 41, 50, 55, 60, 63 are not matched. Is it correct that empty `current_rules` are considered that the rule is found? If not, it could be simplified as:

```
found = False
    for rule in current_rules:
        ￼if rule.get('src') == 'all' or rule.get('dst') == 'all':
             continue
        ...
       if ip_from and ...:
           continue
       ...
       if table is not None and ...:
           continue
       found = True
       break
```

IMHO it would be even nicer to write the loop with `while not found` since it would directly describe its purpose, for example like:

```
current_rules = iter(json.load(result[1])
found = False
while not found:
    rule = next(current_rules, None)
    if not rule:
        break
    if rule.get...
   ...
   if table is not None
   found = True
```

What is your opinion?

Reply:
Have changed to start the loop initially with `found = False`.